src/ajax/xhr.js

```diff
@@ -15,7 +15,6 @@ var xhrSuccessStatus = {
 jQuery.ajaxTransport( function( options ) {
 	var callback;
 
-	// Cross domain only allowed if supported through XMLHttpRequest
```

Comment:
This is unrelated but this comment has long been invalid… We only have support checks on `3.x-stable` here.